test/JsonUtilTest.cpp

```diff
@@ -12,6 +12,14 @@
 #include "util/json.h"
 
 TEST(JsonUtilityFunctionTests, fileToJson) {
+  // Creates a file with the given name and content. Can be deleted with
+  // `ad_utility::deleteFile(fileName)`.
+  auto createFile = [](std::string_view fileName, auto fileContent) {
+    std::ofstream tempStream{ad_utility::makeOfstream(fileName)};
+    tempStream << fileContent << std::endl;
+    tempStream.close();
```

Comment:
close is not needed, the destructor does that for us.

Reply:
But isn't it easier to read like this and more obvious?

Reply:
Well, 
In generally programmers should know that `ofstream` is an RAII type that does the right thing in its destructor, but I don't care too much in this concrete case.